e2e/tests/workspace/workspace-ui.spec.ts

```diff
@@ -1,9 +1,9 @@
-import BaseElement from '../../app/aou-elements/base-element';
-import {SideNavLink} from '../../app/authenticated-page';
-import HomePage from '../../app/home-page';
-import WorkspaceCard from '../../app/workspace-card';
-import WorkspacesPage from '../../app/workspaces-page';
-import {signIn} from '../app';
+import BaseElement from 'app/aou-elements/base-element';
+import {SideNavLink} from 'app/authenticated-page';
+import HomePage from 'app/home-page';
+import WorkspaceCard from 'app/workspace-card';
+import WorkspacesPage from 'app/workspaces-page';
+import {signIn} from 'tests/app';
```

Comment:
Is there a way to set it up so that there's only one `app` directory?

Reply:
are you talking about `tests/app`? it is a file `app.ts`. I see now filename and path `app` are confusing. I'll rename file `app.ts` in future PR. thank you.

Reply:
Yes, that makes sense.